src/hooks/useCroct.ssr.test.tsx

```diff
@@ -1,12 +1,17 @@
 import {renderHook} from '@testing-library/react-hooks/server';
 import {ReactNode} from 'react';
+import {Plug} from '@croct/plug';
 import {useCroct} from './useCroct';
 import {CroctProvider} from '../CroctProvider';
 
+type RenderHookProps = {
+    children: ReactNode,
+};
+
 describe('useCroct', () => {
     it('should not fail on server-side rendering', () => {
-        const {result} = renderHook(() => useCroct(), {
-            wrapper: ({children}: {children?: ReactNode}) => (
+        const {result} = renderHook<RenderHookProps, Plug>(() => useCroct(), {
```

Comment:
Shouldn't this be `CroctProviderProps`?

```suggestion
        const {result} = renderHook<CroctProviderProps, Plug>(() => useCroct(), {
```

Reply:
Nice!